### DISCUSSION TITLE
Share blog post and video tutorials

### DISCUSSION THREAD

**Discussion by bluebill1049** [react-hook-form/react-hook-form#4602] [General]
If you would like to include your blog post or tutorial to be a part of the resource: 

# V7

Post a comment with the following data:

```
{
    "title": "React Hook Form V7 (Preview)",
    "url": "https://www.youtube.com/watch?v=ZBfBiwyR2HY",
    "authorUrl": "https://www.youtube.com/channel/UC9qgXurGpQzePCx928oyIZg",
    "author": "Bill Luo",
    "type": "article", // or "video"
    "description": "Introduction to React Hook From V7 (preview). Hope you will enjoy the next update",
}
```

# V6

You can fork the repo below and send us a PR:

https://github.com/react-hook-form/documentation/edit/master/src/data/resources.tsx

**Comment by jorisre** [comment]
@bluebill1049 Where can I see V6 resources ? 
It only displays v7 resources: https://react-hook-form.com/resources

**Reply by bluebill1049** [reply]
oops... it's a bug...

**Reply by bluebill1049** [reply]
fixed.
